.gitmodules

```diff
@@ -4,3 +4,7 @@
 [submodule "xpring-common-js"]
 	path = xpring-common-js
 	url = git@github.com:xpring-eng/xpring-common-js.git
+[submodule "rippled"]
+	path = rippled
+	url = https://github.com/ripple/rippled
+	branch = develop
```

Comment:
Just double-checking that `develop` is the correct branch here? I'm guessing the protos haven't made their way to master yet in rippled, so this is just temporary, yes?

Reply:
Right now the plan is to use develop (Confirmed with @cjcobb23). In the future, we may consider using checkpoints, but since things should be backwards compatible it's relatively safe to just pull from head.